akka-actor/src/main/resources/reference.conf

```diff
@@ -420,6 +420,7 @@ akka {
         mailbox-type = "akka.dispatch.UnboundedMailbox"
       }
 
+      # deprecated (because relies on blocking)
```

Comment:
Hm... I thought of that initially but seems I started from the wrong side (from inside the mailboxes), checking the timeout could be done while creating the mailbox... I'll give it a shot